fusor-ember-cli/app/controllers/subscriptions/review-subscriptions.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import NeedsDeploymentMixin from "../../mixins/needs-deployment-mixin";
+
+export default Ember.Controller.extend(NeedsDeploymentMixin, {
+
+  isDisconnected: Ember.computed.alias('deploymentController.isDisconnected'),
+
+  backRouteNameReviewSubs: Ember.computed('isDisconnected', function() {
+    if (this.get('isDisconnected')) {
+        return 'subscriptions.credentials';
+    } else {
+        return 'subscriptions.select-subscriptions';
+    }
+  }),
+
+  sortProps: ['contract_number'],
+  sortedModel: Ember.computed.sort('model', 'sortProps'),
```

Comment:
sorted here by `contract_number` since for some reason if showed in reverse order than on the `select-subscriptions` tab